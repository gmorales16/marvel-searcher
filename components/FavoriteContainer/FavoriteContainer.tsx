import { Container, ContainerCard } from "./styledFavoriteContainer";

const FavoriteContainer = () => {
  return (
    <>
      <Container data-testid="favorite-container">
        <ContainerCard data-testid="favorite-card-container"></ContainerCard>
      </Container>
    </>
  );
};

export default FavoriteContainer;
