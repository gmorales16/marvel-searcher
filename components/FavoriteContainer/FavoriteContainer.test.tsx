import "jsdom-global/register";
import "@testing-library/jest-dom/extend-expect";
import React from "react";
import { render, screen } from "@testing-library/react";
import FavoriteContainer from "./FavoriteContainer";

describe("<FavoriteContainer/>", () => {
  test("renders FavoriteContainer Component", () => {
    render(<FavoriteContainer />);
    const FavoriteContainerElement =
      screen.getAllByTestId("favorite-container");
    const FavoriteCardContainerElement = screen.getAllByTestId(
      "favorite-card-container"
    );
    expect(FavoriteContainerElement[0]).toBeInTheDocument();
    expect(FavoriteCardContainerElement[0]).toBeInTheDocument();
    // Take a snapshot of the SearchBar component
    const { asFragment } = render(<FavoriteContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
