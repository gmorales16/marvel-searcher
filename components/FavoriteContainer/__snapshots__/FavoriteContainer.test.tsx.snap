// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoriteContainer/> renders FavoriteContainer Component 1`] = `
<DocumentFragment>
  <div
    class="FavoriteContainer__Container-sc-304lf1-0 dspwId"
    data-testid="favorite-container"
  >
    <div
      class="FavoriteContainer__ContainerCard-sc-304lf1-1 kqdimq"
      data-testid="favorite-card-container"
    />
  </div>
</DocumentFragment>
`;
